docs/design/features/runtime-handled-tasks.md

```diff
@@ -305,20 +305,20 @@ static Continuation? IL_STUB_AsyncResume_Foo(Continuation continuation)
 
 ### Intrinsics used for suspension/resumption
 
-To interact with the async2 calling convention the JIT/VM provides the following intrinsics:
+To interact with the async calling convention the JIT/VM provides the following intrinsics:
 ```csharp
-// Retrieve the continuation returned by a preceding async2 function call
+// Retrieve the continuation returned by a preceding async function call
 [Intrinsic]
-internal static Continuation? Async2CallContinuation() => null;
+internal static Continuation? AsyncCallContinuation() => null;
 
 // Suspend the current function by immediately returning with a specific non-zero continuation.
 [Intrinsic]
-private static void SuspendAsync2(Continuation continuation) => throw new UnreachableException();
+private static void SuspendAsync(Continuation continuation) => throw new UnreachableException();
```

Comment:
NIT: Maybe `AsyncSuspend` is a better name, since we are changing the name anyways.

Reply:
Renamed